docs/charts.md

```diff
@@ -158,7 +158,8 @@ the preferred method of declaring dependencies is by using a
 `requirements.yaml` file inside of your chart.
 
 **Note:** The `dependencies:` section of the `Chart.yaml` from Helm
-Classic has been completely removed.
+Classic has been completely removed.  Dependencies are now specified
```

Comment:
I don't think there is a need to add the sentence starting with "Dependencies are now specified ...." as this is mentioned above the note and the note is specific to the Helm classic point.